common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1314,156 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	stacking = yes
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+		heavy_infantry = 0.05
+		light_infantry = 0.05
+		archers = 0.05
+		light_cavalry = 0.05
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion = fel_group
+			}
+		}
+	}
+
+	stacking = yes
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -5
+	death_god_opinion = 5
+	command_modifier = {
+		morale_defence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_scourge_bone_chimes"
+	
+	slot = leader
+}
+frost_wyrm_skull_shield = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+
+	combat_rating = 20
+
+	monthly_character_prestige = 0.75
+	
+		command_modifier = {
+		morale_defence = 0.05
+		defence = 0.05
+	}
+
+	picture = "GFX_frost_wyrm_skull_shield"
+	slot = accessoire
+}
+
+# Uncommon
+voodoo_doll = {
+	quality = 2
+	flags = { misc uncommon }
+
+	active = {
+		is_adult = yes	
+	}
+
+	stacking = no
+	indestructible = yes
+
+	intrigue = 1
+	learning = 1
+
+	monthly_character_prestige = 0.1
+	monthly_character_piety = 0.2
+	
+	same_religion_opinion = 5
```

Comment:
So, holy light character with voodoo doll will be more respectable by other holy light characters than holy light character without voodoo doll?

Reply:
should I limit to loa priests?

Reply:
loa-priest only